src/Symfony/Component/Console/Command/Command.php

```diff
@@ -447,6 +448,26 @@ public function getName()
     }
 
     /**
+     * @param bool $public Whether the command should be publicly shown or not.
+     * 
+     * @return Command The current instance
+     */
+    public function setPublic($public)
```

Comment:
I suggested changing it to `setPublic(false)` to mimic Symfony services ... where you must set `public: false` for private services. If we use opposing techniques to do the same (`public: false` for services, `private: true` for commands) we're increasing the learning curve.